lib/kitchen/driver/opennebula.rb

```diff
@@ -65,6 +65,11 @@ def initialize(config)
       def create(state)
         conn = opennebula_connect
 
+        # Check for servers from connection to help debug possible connection issues
+        if conn.servers.length == 0
```

Comment:
Hi @Gnof, this check will also return 0 if the user legitimately has 0 virtual machines running under OpenNebula.  Using the OpenNebula get_version, and checking its return code should be a more reliable way of checking whether the user is authenticated.